login-workflow/docs/existing-project-migration.md

```diff
@@ -0,0 +1,135 @@
+# Update React Auth Workflow Package
+
+Update @brightlayer-ui/react-auth-workflow package to latest version.
+```shell
+npm install --save @brightlayer-ui/react-auth-workflow
+// or
+yarn upgrade @brightlayer-ui/react-auth-workflow
+```
+
+## Remove @brightlayer-ui/react-auth-shared package
+
+In latest version we have removed @brightlayer-ui/react-auth-shared.
+
+## Update AuthUIActions
+
+Since we have removed `@brightlayer-ui/react-auth-shared` package `SecurityContextActions` are not supported by latest workflow. Instead of this you need to manage your app's state by app context. 
+
+For e.g You can replace SecurityContextActions with `AppContextType` which you can declare in AppContextProvider. You can check [here](https://github.com/etn-ccis/blui-react-workflows/blob/dev/login-workflow/example/src/contexts/AppContextProvider.tsx) for detailed implementation.
```

Comment:
The link should point to master branch, instead of dev (for now, it will be broken)